### ISSUE TITLE
new update thunderbird 128 - xnote fail

### ISSUE THREAD

**Issue by zuzana25** [xnotepp/xnote#138]
windows 10 (64 b vers), 
after update thunderbird 115 to thunderbird 128
all the notes on the emails are gone, how can I view them

![xnotes fail](https://github.com/user-attachments/assets/fcd19b0c-7d53-4551-bee4-b5bf76b560a6)

**Comment by rieschl** [comment]
FYI: I tried to download the [lastest release](https://github.com/xnotepp/xnote/releases/tag/v4.2.0), extracted the file (it's just a zip file) and then changed `strict_max_version` in `manifest.json` to `1.128.*`. After that, I zipped everything again and installed the addon in Thunderbird via `install from file`.
It seems to be working 🙂

**Comment by Chealer** [comment]
@zuzana25 : Please indicate which version you use and retitle to something like:

> All notes disappeared following upgrade to Thunderbird 128

**Comment by eddieturtle** [comment]
> FYI: I tried to download the [lastest release](https://github.com/xnotepp/xnote/releases/tag/v4.2.0), extracted the file (it's just a zip file) and then changed `strict_max_version` in `manifest.json` to `1.128.*`. After that, I zipped everything again and installed the addon in Thunderbird via `install from file`. It seems to be working 🙂

Tried this and it gave an error saying it's not compatible with this version of Thurderbird

**Comment by opto** [comment]
yes, I am working on this. On TB start, there can be timing problems, and it works for some users, but not all. ut we will fix that.

Klaus

**Comment by zuzana25** [comment]
> @zuzana25 : Please indicate which version you use and retitle to something like:
> 
> > All notes disappeared following upgrade to Thunderbird 128

verzia 4.1.12

**Comment by opto** [comment]
see https://github.com/xnotepp/xnote/releases/tag/v4.3.24

**Comment by Chealer** [comment]
> [@zuzana25](https://github.com/zuzana25) : Please indicate which version you use and retitle to something like:
> 
> > All notes disappeared following upgrade to Thunderbird 128

I recommend to mark this report as incomplete or somehow archive it.

**Comment by opto** [comment]
In any case, even if not displayed, the XNote files should be in the XNote folder in the profile or in the location that you set in options.
